integration/test/Test/Channels.hs

```diff
@@ -590,3 +590,18 @@ testAdminCanRemoveMemberWithoutJoining = do
       void $ mlsCliConsume convId def cid msgData
       r <- createPendingProposalCommit convId cid >>= sendAndConsumeCommitBundle
       shouldBeEmpty $ r %. "events"
+
```

Comment:
We should also add a test to ensure that admins can't get non-channel conversations.